aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -100,7 +100,7 @@ impl<'e, E: ExecutorView> StorageAdapter<'e, E> {
             self.accurate_byte_count = true;
         }
         self.max_binary_format_version =
-            get_max_binary_format_version(features, gas_feature_version);
+            get_max_binary_format_version(features, Some(gas_feature_version));
```

Comment:
We constantly improve the code while we are visiting it. Here I was improving the way how the the current default binary format version was determined, since I'm adding a new version. There were multiple places how the default was retrieved ad hoc, and it took me some to figure this out, so I unified them into this function here. However, for the unification I needed to make the 2nd parameter optional.